templates/include/telemetry/environment.1.schema.json

```diff
@@ -451,90 +447,81 @@
-                    },
-                    "driverVersion": {
-                      "type": [ "string", "null" ]
-                    },
-                    "driverDate": {
-                      "type": [ "string", "null" ]
-                    },
-                    "GPUActive": {
-                      "type": "boolean"
-                    }
+                "type": "object",
+                "properties": {
+                  "description": {
+                    "type": [ "string", "null" ]
+                  },
+                  "vendorID": {
+                    "type": [ "string", "null" ]
+                  },
+                  "deviceID": {
+                    "type": [ "string", "null" ]
+                  },
+                  "subsysID": {
+                    "type": [ "string", "null" ]
+                  },
+                  "RAM": {
+                    "type": [ "integer", "null" ]
+                  },
+                  "driver": {
+                    "type": [ "string", "null" ]
+                  },
+                  "driverVersion": {
+                    "type": [ "string", "null" ]
+                  },
+                  "driverDate": {
+                    "type": [ "string", "null" ]
+                  },
+                  "GPUActive": {
+                    "type": "boolean"
                   }
                 }
               }
             },
             "monitors": {
-              "type": [
-                "array",
-                "object"
-                ],
+              "type": ["array", "object"],
               "items": {
-                "monitor": {
-                  "type": "object",
-                  "properties": {
-                    "screenWidth": {
-                      "type": "integer"
-                    },
-                    "screenHeight": {
-                      "type": "integer"
-                    },
-                    "refreshRate": {
-                      "type": "number"
-                    },
-                    "pseudoDisplay": {
-                      "type": "boolean"
-                    },
-                    "scale": {
-                      "type": "number"
-                    }
+                "type": "object",
+                "properties": {
+                  "screenWidth": {
+                    "type": "integer"
+                  },
+                  "screenHeight": {
+                    "type": "integer"
+                  },
+                  "refreshRate": {
+                    "type": "number"
+                  },
+                  "pseudoDisplay": {
+                    "type": "boolean"
+                  },
+                  "scale": {
+                    "type": "number"
                   }
                 }
               }
             },
             "features": {
               "type": "object",
-              "items": {
-                "feature": {
-                  "type": "object",
```

Comment:
Sigh, this section never matched reality https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/environment.html